src/Type/JsonDocumentType.php

```diff
@@ -109,8 +109,8 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
-        if (null === $value) {
-            return;
+        if (null === $value || $value === '') {
+           return array();
```

Comment:
`null` is valid JSON isn't it? IMO we should return `null` instead of `[]` in this case.

Reply:
https://3v4l.org/TilkM
json_decode(null) throw an exception

Reply:
It will not be called right?